pkg/kf/commands/quotas/update_test.go

```diff
@@ -68,6 +70,56 @@ func TestUpdateQuotaCommand(t *testing.T) {
 					Return(nil)
 			},
 		},
+		"update success": {
+			args: []string{"some-quota", "-m", "20Gi"},
+			setup: func(t *testing.T, fakeUpdater *fake.FakeClient) {
+				fakeUpdater.
+					EXPECT().
+					Transform(gomock.Any(), gomock.Any(), gomock.Any()).
+					Do(func(namespace string, name string, transformer quotas.Mutator) error {
+						kfquota := quotas.NewKfQuota()
+						memQuantity, err := resource.ParseQuantity("1024M")
```

Comment:
You should probably use a testutil.AssertNil to validate this error and the one on line 82 _or_ ignore them both (preferably the former to make sure the test doesn't fail if ParseQuantity changes behavior)

Reply:
done, also moved out creating the test quota into a separate func